src/components/organisms/topbar/project/projectConfig.tsx

```diff
@@ -19,17 +19,27 @@ import {
 export const ProjectConfigTopbar = ({ variant }: { variant: TopbarType }) => {
 	const { projectId } = useParams();
 	const { openProjectId, setOpenProjectId } = useFileOperations(projectId!);
-	const { fetchDeployments } = useCacheStore();
+	const { fetchDeployments, initCache } = useCacheStore();
 
 	useEffect(() => {
-		if (!projectId || variant !== TopbaButtonVariant.actions) return;
+		if (!projectId) return;
+
+		initCache(projectId, true);
```

Comment:
We don't need to init cache in case we're on sessions page.
We don't need an assets cache on the deployments page.
We don't need a deployments cache on the assets page.

Reply:
done